### DISCUSSION TITLE
使用多个吸盘

### DISCUSSION THREAD

**Discussion by wh201906** [dukelec/cd_pnp#11] [Ideas]
可以同时使用多个气泵和吸盘吗？这样就能在单次移动中贴多个相同或者等高的元器件了

**Comment by dukelec** [comment]
谢谢建议，为了保持格价便宜和保持机器小巧，暂时不考虑增加吸嘴。

如果板子数量相对较多，想提高速度，可以考虑搞多台机器并行作业。
